### DISCUSSION TITLE
Add Color Profile Option in System Settings

### DISCUSSION THREAD

**Discussion by Mashrafi2006** [elementary/triage#211] [Ideas]
As a user of Elementary OS, I have noticed that there is no option to change the color profile of the monitor. This can cause the colors to look off or inaccurate, especially for users who work with digital media. Therefore, I would like to suggest adding a color profile option in the System Settings.

This option could be located in the Displays section of the System Settings. When selected, the user could choose from a list of pre-installed color profiles or import their own. This would allow users to calibrate their monitors and ensure accurate color representation.

I believe this feature would be a valuable addition to Elementary OS and would greatly improve the user experience for those who require accurate color representation. Thank you for considering my suggestion.

**Comment by danirabbit** [comment]
Hey thanks for opening this discussion this is actually a duplicate of a reported issue that's being tracked here: https://github.com/elementary/switchboard-plug-display/issues/5
